src/Pages/Login/Login.js

```diff
@@ -1,7 +1,58 @@
 import React, { Component } from 'react';
+import LoginBtn from './LoginBtn';
+import LoginInput from './LoginInput';
+import './Login.scss';
+import LoginTextPage from './LoginTextPage';
 
 export default class Login extends Component {
+  state = {
+    idValue: '',
+    passwordValue: '',
+  };
+  handleIdChange = e => {
+    this.setState({
+      idValue: e.target.value,
+    });
+  };
+  handlePasswordChange = e => {
+    this.setState({
+      passwordValue: e.target.value,
+    });
+  };
```

Comment:
```suggestion
  };
  
  handleIdChange = e => {
    this.setState({
      idValue: e.target.value,
    });
  };
  
  handlePasswordChange = e => {
    this.setState({
      passwordValue: e.target.value,
    });
  };
```
- 구분되는 내용들 끼리는 한줄 띄워주세요
- 객체 계산된 속성명 문법을 사용하면 두 함수를 하나로 합쳐줄 수 있을 것 같네요. 다음 문서 참고해주세요. 방법 고민해보시고 정 모르시겠으면 질문주세요!
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Object_initializer#%EA%B3%84%EC%82%B0%EB%90%9C_%ED%94%84%EB%A1%9C%ED%8D%BC%ED%8B%B0%EB%AA%85